filters/auth/forwardtokenfield.go

```diff
@@ -0,0 +1,89 @@
+package auth
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/filters"
+	"golang.org/x/net/http/httpguts"
+)
+
+const (
+	ForwardTokenFieldName = "forwardTokenField"
+)
+
+type (
+	forwardTokenFieldSpec   struct{}
+	forwardTokenFieldFilter struct {
+		HeaderName string
+		Field      string
+	}
+)
+
+// NewForwardTokenPart creates a filter to forward fields from token info or
+// token introspection or oidc user info as headers to the backend server.
+func NewForwardTokenField() filters.Spec {
+	return &forwardTokenFieldSpec{}
+}
+
+func (s *forwardTokenFieldSpec) Name() string {
+	return ForwardTokenFieldName
+}
+
+func (*forwardTokenFieldSpec) CreateFilter(args []interface{}) (filters.Filter, error) {
+	if len(args) != 2 {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+
+	headerName := args[0].(string)
```

Comment:
```go
headerName, ok := args[0].(string)
if !ok {
   return nil, filters.ErrInvalidFilterParameters
}
```
the same is for the second arg.

The https://github.com/zalando/skipper/pull/1814 targets to eliminate such boilerplate

Reply:
@AlexanderYastrebov but #1814 is not merged to master yet, how can I use it in my PR?

Reply:
It is just a note that hopefully in the future we'll get rid of the boilerplate, please use `headerName, ok := args[0].(string)` for now.